### PR TITLE
Display number of likes for each item on homepage

### PR DESCRIPTION
**Pull request by Timbar09**
I applied functionality to display the likes count for each item in this PR.

**Changes**

- Implemented a function for rendering the likes count for each item to the DOM
- Updated the renderShowsToDOM function
- Added styling to the items and the likes count

### CODE REVIEW

File: src/modules/renderToDOM.js [hunk 1]
```diff
@@ -1,24 +1,49 @@
...
 
+export const renderLikesToDOM = async () => {
+  try {
+    const response = await fetch(`${INVOLVEMENT_API_URL}apps/${appID}/likes/`);
+
+    const data = await response.json();
+
+    return data;
+  } catch (err) {
+    throw new Error('Error fetching item likes');
```
Comment (oyhpnayiaw) [inline src/modules/renderToDOM.js:22]: This catch block will always run if `appID` is new. So instead of throwing new error just return the empty array.

**Review by oyhpnayiaw — CHANGES_REQUESTED** [review]
Good job on implementing number of likes for each item on home page.  👍 

But there is something you need to change. 
- Please make sure all linters passed.
- See the comments extra informations. 

[Optional Suggestion] - Link your PR with the issue that way the issue will also mark as completed when PR is merged.

File: src/modules/renderToDOM.js (hunk 1 above)
Comment (Timbar09) [reply src/modules/renderToDOM.js:22]: Ok, I got you.

Comment (Timbar09) [reply src/modules/renderToDOM.js]: I made the changes, bro.

Comment (oyhpnayiaw) [reply src/modules/renderToDOM.js]: Hello @Timbar09. Yeah I see bro but the conflicts need to resolve to merge bro.

**Pull request merged**